GooglePlayInstant/Editor/PlayInstantQuickDeployHttpRequestHelper.cs

```diff
@@ -0,0 +1,108 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using UnityEngine;
+
+[assembly: InternalsVisibleTo("GooglePlayInstant.Tests.Editor.QuickDeploy")]
+
+namespace GooglePlayInstant.Editor
+{
+    /// <summary>
+    /// A class with utility methods for sending GET and POST requests.
```

Comment:
You might mention that you're using the older WWW style objects for backwards compatibility with older versions of Unity.

Reply:
Resolved.